packages/web-components/fast-foundation/src/tree-view/tree-view.template.ts

```diff
@@ -14,6 +14,6 @@ export const TreeViewTemplate = html<TreeView>`
         @focus="${(x, c) => x.handleFocus(c.event as FocusEvent)}"
         @blur="${(x, c) => x.handleBlur(c.event as FocusEvent)}"
     >
-        <slot ${slotted("slottedTreeItems")}></slot>
```

Comment:
I think we still need a default slot for things which are not items. I brought up the divider example primarily because I know it's an existing use case for a few folks. I don't think it's a requirement for tree items, but it is one for tree view. Dividers are passed to break up the tree view visually but it's still part of the same keyboarding/interaction model. Example:

![image](https://user-images.githubusercontent.com/13071055/87341094-de166100-c4fd-11ea-9dc3-e969a2ea8f8e.png)

Reply:
This shows a single instance, but there may be multiples and their placement is fairly arbitrary. I'm not sure how we would slot this aside from adding a slot of "item", which doesn't seem semantically correct as it's not a tree item - but that seems to be the only way to get it to slot in order given the API change here. I'm good with the item portion of this, but unsure how it would continue to support this type of behavior. Perhaps @EisenbergEffect has a suggestion on how we could accommodate this outside of using a default slot for these or adding `slot="item"` to this?

Reply:
Perhaps I could suggest multiple tree views at that point, so:
```html
<fast-tree-view>
    <fast-tree-item>Foo</fast-tree-item>
</fast-tree-view>
<fast-divider />
<fast-tree-view>
    <fast-tree-item>Bar</fast-tree-item>
</fast-tree-view>
```

Reply:
I don't think that would meet the requirements of having it navigate within the same tree construct - I'd need to move from one to the other via tab which isn't the intended behavior and may be confusing to users.